Project.toml

```diff
@@ -67,6 +67,7 @@ DomainSets = "5b8099bc-c8ec-5219-889f-1d9e522a28bf"
 Graphviz_jll = "3c863552-8265-54e4-a6dc-903eb78fde85"
 HomotopyContinuation = "f213a82b-91d6-5c5d-acf7-10f1c761b327"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
Logging is required to turn of logging in StructuralIdentifiability. I would suggest to them to not require it to be explicitly loaded, but I feel a bit bad to give further comments on how they set it up (especially since I was the one who suggested they add an option to remove logging in the first place). Since Logging is a base package I am not to worried about adding it here, but removing it is no problem either.